pkg/lib/clusterconfig/clusterconfig.go

```diff
@@ -21,42 +21,48 @@ import (
 
 	"github.com/cortexlabs/cortex/pkg/consts"
 	"github.com/cortexlabs/cortex/pkg/lib/aws"
+	"github.com/cortexlabs/cortex/pkg/lib/table"
+	kresource "k8s.io/apimachinery/pkg/api/resource"
+
```

Comment:
nit: organize imports (side note: I found that if you remove all new lines and then `gofmt`, it will add the new line in the right place)